tests-legacy/Integration/PrestaShopBundle/Controller/Api/Improve/Design/PositionsControllerTest.php

```diff
@@ -51,16 +51,17 @@ public function setUp()
 
         /** @var ModuleManager */
         $moduleManager = self::$kernel->getContainer()->get('prestashop.module.manager');
-        if (!$moduleManager->isInstalled('bankwire')) {
-            $moduleManager->install('bankwire');
+
+        if (!$moduleManager->isInstalled('ps_banner')) {
```

Comment:
trolololo

Reply:
Why change the module in question?

Reply:
I struggled several hours to fix PositionsControllerTest::testMoveHookPositionToTop . Looks like `ps_featuredproducts` is in a strange state: it is installed according to ModuleManager but its hooks are not registered. Tried uninstall and install again, did not work.

I replaced the use of `ps_featuredproducts` (the git diff makes it hard to see but if you compare old/new version it's obvious) for this test by `ps_banner` and it solves the issue.

I think something is wrong with this module, maybe another test modified it / installed it and the fixture state is corrupted.